test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetToolTests.cs

```diff
@@ -675,7 +674,7 @@ public async Task DotnetToolTests_IncompatibleAutorefPackageAndToolsPackageAsync
 
                 _msbuildFixture.CreateDotnetToolProject(solutionRoot: testDirectory.Path,
                     projectName: projectName, targetFramework: tfm, rid: rid,
-                    source: source, packages: packages);
+                    packageSource: source, packages: packages);
```

Comment:
All these package source change seem unnecessary. 

I'm not sure the extra qualifier adds much to the method.

Reply:
Just making it clear what `source` means here. It's public method that we are using it to write tests. So, it should be clear on what `source` is.